consensus/polybft/consensus_runtime_test.go

```diff
@@ -34,28 +34,27 @@ func TestConsensusRuntime_GetVotes(t *testing.T) {
 	)
 
 	validatorIds := []string{"A", "B", "C", "D", "E", "F", "G"}
-	validatorAccounts := newTestValidatorsWithAliases(validatorIds)
+	validators := newTestValidatorsWithAliases(validatorIds)
+	validatorAccs := validators.getPublicIdentities()
 	state := newTestState(t)
 	runtime := &consensusRuntime{
 		state: state,
 		epoch: &epochMetadata{
 			Number:     epoch,
-			Validators: validatorAccounts.getPublicIdentities(),
+			Validators: validatorAccs,
 		},
 	}
 
 	commitment, _, _ := buildCommitmentAndStateSyncs(t, stateSyncsCount, epoch, 0)
 
-	quorumSize := validatorAccounts.toValidatorSetWithError(t).getQuorumSize()
-
 	require.NoError(t, state.insertEpoch(epoch))
 
-	votesCount := quorumSize + 1
+	votesCount := int(getQuorumSize(validatorAccs.GetTotalVotingPower()).Int64())
```

Comment:
Why has it changed?

Reply:
It is changed because of changes made to the `validatorSet`. I haven't really felt like `getQuroumSize` should be tight with validator set instance, because it requires only total voting power.

Btw entire `validatorSet` struct feels like a bit redundant and is there only because `AccountSet` is a type alias rather than a proper struct. Maybe we can discuss it in the future to merge `validatorSet` and `AccountSet` into a single implementation.